<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EllipseDetectionDialog</class>
 <widget class="QDialog" name="EllipseDetectionDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>211</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Ellipse Detection</string>
  </property>
  <property name="windowOpacity">
   <double>0.800000000000000</double>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>9</y>
     <width>381</width>
     <height>191</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="4" column="0">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Max Diameter</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0" colspan="3">
     <widget class="QCheckBox" name="activateBox">
      <property name="text">
       <string>Activate Processing</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Threshold</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0" colspan="3" alignment="Qt::AlignHCenter">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Fit parameters</string>
      </property>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="QSlider" name="MinDiameter">
      <property name="value">
       <number>50</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="5" column="0" colspan="3">
     <widget class="QPushButton" name="feedbackButton">
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="text">
       <string>Show Edges</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="4" column="2">
     <widget class="QSlider" name="MaxDiameter">
      <property name="value">
       <number>50</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Min Diameter</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1" colspan="2">
     <widget class="QSlider" name="thresholdSlider">
      <property name="pageStep">
       <number>1</number>
      </property>
      <property name="value">
       <number>50</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
